client/app/components/service-explorer/service-explorer.html

```diff
@@ -0,0 +1,250 @@
+           class="service-explorer-children-list"
+           config="vm.serviceChildrenListConfig"
+           items="$parent.item.all_service_children"
+           custom-scope="vm"
+           update-menu-action-for-item-fn="vm.updateMenuActionForItemFn">
+        <div class="row">
+          <div class="col-lg-4 col-md-4 col-sm-5 col-xs-8 name-column">
+            <span class="no-wrap">
+              <i class="fa pficon fa-power-off" ng-if="item.power_state === 'off'"
+                 uib-tooltip="{{'Power State: Off'|translate}}"
+                 tooltip-append-to-body="true"
+                 tooltip-popup-delay="1000"
+                 tooltip-placement="bottom"></i>
+              <i class="pficon pficon-ok" ng-if="item.power_state === 'on'"
+                 uib-tooltip="{{'Power State: On'|translate}}"
+                 tooltip-append-to-body="true"
+                 tooltip-popup-delay="1000"
+                 tooltip-placement="bottom"></i>
+              <i class="pficon fa fa-question-circle" ng-if="item.power_state !== 'on' && item.power_state !== 'off'"
+                 uib-tooltip="{{'Power State: Unknown'|translate}}"
+                 tooltip-append-to-body="true"
+                 tooltip-popup-delay="1000"
+                 tooltip-placement="bottom"></i>
+              <a uib-tooltip="{{item.name}}"
+                 tooltip-placement="bottom"
+                 tooltip-append-to-body="true"
+                 tooltip-popup-delay="1000"
+                 ng-click="customScope.viewService(item, $event)">
+              <span class="ss-list-view__title-img">
+                <span class="ss-list-view__title-img__center"></span>
+                <img class="ss-list-view__title-img__logo" ng-src="{{ item.picture.image_href }}"
+                     ng-if=" item.picture.image_href"/>
+                <img class="ss-list-view__title-img__logo" src="images/service.png" ng-if=" !item.picture.image_href"/>
+              </span>
+              <span>
+                {{ item.name }}
+              </span>
+              </a>
+            </span>
+          </div>
+          <div class="col-lg-3 col-md-3 col-sm-3 col-xs-4">
+          </div>
+          <div class="col-lg-2 col-md-2 col-sm-2 hidden-xs">
+            <div class="list-view-stacked-item" uib-tooltip="Created: {{item.created_at | date : 'MMM dd, yyyy H:mm:ss'}}"
+                 tooltip-append-to-body="true"
+                 tooltip-popup-delay="1000"
+                 tooltip-placement="bottom">
+              <strong>Created</strong>&nbsp;
+              <div>
+                {{ item.created_at | date : 'shortDate' }}
+              </div>
+            </div>
+          </div>
+          <div class="col-lg-2 col-md-2 col-sm-2 hidden-xs">
+            <div class="list-view-stacked-item"
+                 uib-tooltip="{{item.retires_on ? (item.retires_on | date : 'medium') : ('Never' | translate)}}"
+                 tooltip-append-to-body="true"
```

Comment:
Ah append-to-body yeah, good call on that, tooltip placement though is set to `top` by default, more places than not we use it on the bottom though, I would think it to would make sense to make the default whatever we use more often throughout the app and override when necessary. 

Will make a pt encompassing this work and start on it today (if you're on board and nothing else flares up)